### PR TITLE
{bio}[foss/2020b] QUAST v5.0.2 w/ Python 2.7.18

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#12718~~ (matplotlib)

### CODE REVIEW

File: easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2020b-Python-2.7.18.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    ('Java', '11', '', True),
+    ('Boost', '1.74.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('simplejson', '3.17.2', {
+        'checksums': ['75ecc79f26d99222a084fbdd1ce5aad3ac3a8bd535cd9059528452da38b68841'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2020b-Python-2.7.18.eb:28]: This is no longer needed, as it is provided by our Python 2.7.18 install: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-10.2.0.eb#L382

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3604.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/b6e692254f22931a0a5b14cd454b65c5 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/a3df7757c5275420e77fb06095daae60 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**